ipmi/MANIFEST

```diff
@@ -119,5 +114,14 @@
 		}
 	],
 	"Dependencies": null,
-	"ConfigMacros": null
+	"ConfigMacros": [
+		{
+                        "MacroName": "IPMI",
+                        "Description": "IPMI tag name",
+                        "DefaultValue": "ipmi",
+                        "Value": "",
+                        "Type": "",
```

Comment:
The IPMI issue referenced syslog as an example. In that example: `"Type": "TAG",`
This is also for a tag, so I think it would be appropriate to add that here.